client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,94 @@
+# KServe API usage samples
+
+OpenVINO Model Server 2022.2 release introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+
+## Before you run the samples
+
+### Clone OpenVINO&trade; Model Server GitHub repository and go to the directory with the samples.
+```Bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/client/cpp/kserve-api
+```
+
+### Build client library and samples
+```Bash
+cmake . && make
+```
+
+## GRPC Examples <a name="grpc-api"></a>
+
+### Run the Client to get server liveness <a name="grpc-server-live"></a>
+
+- Command
+
+```Bash
+./grpc_server_live --help
+Sends requests via KServe gRPC API to check if server is alive.
+Usage:
+  grpc_infer_dummy [OPTION...]
+
+  -h, --help              Show this help message and exit
+      --grpc_address arg  Specify url to grpc service.  (default: 
+                          localhost)
+      --grpc_port arg     Specify port to grpc service.  (default: 9000)
+      --timeout arg       Request timeout. (default: 0)
+```
+
+- Usage Example 
+
+```Bash
+./grpc_server_live --grpc_port 9000 --grpc_address localhost
+Server Live: True
+```
+
+## GRPC Examples with Dummy Model
+
+This section demonstrates request to simple model, which increments each provided value. 
+
+### Start the Model Server Container with Dummy Model
+```Bash
+docker run --rm -d -v $(pwd)/src/test/dummy:/models -p 9000:9000 openvino/model_server:latest --model_name dummy --model_path /models --port 9000 
+```
+
+Once you finish above steps, you are ready to run the samples.
+
+### Run the Client to perform inference
+
+```Bash
+./grpc_infer_dummy --help
+Sends requests via KServe gRPC API.
+Usage:
+  grpc_infer_dummy [OPTION...]
+
+  -h, --help              Show this help message and exit
+      --grpc_address arg  Specify url to grpc service.  (default: 
+                          localhost)
+      --grpc_port arg     Specify port to grpc service.  (default: 9000)
+      --input_name arg    Specify input tensor name.  (default: b)
+      --output_name arg   Specify input tensor name.  (default: a)
+      --model_name arg    Define model name, must be same as is in service. 
+                           (default: dummy)
+      --model_version     Define model version.
+      --timeout arg       Request timeout. (default: 0)
+```
+
+- Usage Example
+
+```Bash
+./grpc_infer_dummy --grpc_port 9000 --model_name dummy
+0 => 1
+1 => 2
+2 => 3
+3 => 4
+4 => 5
+5 => 6
+6 => 7
+7 => 8
+8 => 9
+9 => 10
+======Client Statistics======
+completed_request_count 1
+Cumulative total request time 6.70928 ms
+Cumulative send time 0.039253 ms
+Cumulative receive time 0.005686 ms
```

Comment:
I don't know how to interpret those statistics. I think more friendly message could look like:
```
Number of requests: X1
Total processing time: X2
Latency: X3
Requests per second: X4
```